tests/drivers/watchdog/wdt_basic_api/src/test_wdt.c

```diff
@@ -116,6 +116,9 @@
 #define TIMEOUTS 0
 #define WDT_TEST_MAX_WINDOW 3000U
 #endif
+#if DT_HAS_COMPAT_STATUS_OKAY(nxp_wdog32)
+#define WDT_TEST_MAX_WINDOW 1000U
```

Comment:
not required for this PR, but since we're here and this is starting to add up, it seems like this value should come from a test-specific kconfig, it would be a good refactor